cibuild.sh

```diff
@@ -14,7 +14,7 @@ XUNIT_VERSION=2.0.0-alpha-build2576
 BUILD_CONFIGURATION=Debug
 OS_NAME=$(uname -s)
 USE_CACHE=true
-MONO_ARGS='--runtime=v4.0.30319 --gc=boehm --debug=mdb-optimizations --attach=disable'
+MONO_ENV_OPTIONS='--runtime=v4.0.30319 --gc=boehm --debug=mdb-optimizations --attach=disable'
```

Comment:
IIRC: that doesn't set an environment variable, it sets a bash variable.  Need to use `export` to set an ENV variable.